### DISCUSSION TITLE
Adding .mapeoconfig as a format for loading configurations #464

### DISCUSSION THREAD

**Discussion by jencastrodoesstuff** [digidem/mapeo-mobile#805] [Ideas]
As per [issue made in the settings builder](https://github.com/digidem/mapeo-settings-builder/issues/17?_pjax=%23js-repo-pjax-container)

I would like to add the .mapeoconfig extension as an acceptable format for loading configs.  We do have to keep .mapeosettings as a legacy for users with older config packages
 
Having this on the next releases would mean less work of editing materials again after this May deadline,

**Comment by ximenabb** [comment]
@jencastrodoesstuff  we have added https://github.com/digidem/mapeo-settings-builder/issues/17?_pjax=%23js-repo-pjax-container  as a sprint candidate.  is this issue the same?

**Comment by jencastrodoesstuff** [comment]
YES!!!  (Excitement)
